src/components/TransactionsList/TransactionsList.story.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { storiesOf, withInfo } from '../../stories';
 
+import './storybook.css';
```

Comment:
Can this be `TransactionsList.story.css`? To line up with the JS filename name

Reply:
Didn't see this comment until just now, I'll make the change.